exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -1,158 +1,160 @@
-import org.junit.Rule;
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
-import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class TournamentTest {
 
-    @Rule
-    public ExpectedException expectedException = ExpectedException.none();
+    private Tournament tournament;
+
+    @Before
+    public void setUp() throws Exception {
+        tournament = new Tournament();
+    }
 
     @Test
     public void justTheHeaderIfNoInput() {
-        Tournament tournament = new Tournament();
-        assertEquals("Team                           | MP |  W |  D |  L |  P", tournament.printTable());
+        assertEquals("Team                           | MP |  W |  D |  L |  P\n", tournament.printTable());
     }
 
     @Test
-    public void aWinIsThreePointsALossIsZeroPoints() throws Exception {
-        Tournament tournament = new Tournament();
+    @Ignore
```

Comment:
This should have:
```
@Ignore("Remove to run test")
@Test
```

How to add `@Ignore`s is described in the [`Ignore` section of the POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) :)